releases/patch-releases.md

```diff
@@ -47,31 +47,32 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Target date |
 | --- | --- |
-| January 2021 | 2021-01-13 |
```

Comment:
Should we add April instead of January, so we still have 3 items on the list?

Reply:
will add more dates :)

Reply:
done PTAL